### PR TITLE
[RFR] Automate Java Provider

### PR DESCRIPTION
**Pull request by kpunwatk**
Resolves https://issues.redhat.com/browse/MTA-3178 
[Jenkins Run](https://main-jenkins-csb-migrationqe.apps.ocp-c1.prod.psi.redhat.com/view/MTA/job/mta/job/mta-cli-tests-runner/125/console) 
Screenshot from 2024-09-09 14-08-18

### CODE REVIEW

**Comment by kpunwatk** [comment]
Hi @sshveta @nachandr @abrugaro Please Review this PR, Thanks!

File: tests/analysis/java/test_selective_java_provider.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    command = build_analysis_command(
+            application_data['file_name'],
+            "",
+            "eap8",
+            **{'provider': "java"}
+        )
+
+    output = subprocess.run(command, shell=True, check=True, stdout=subprocess.PIPE, encoding='utf-8').stdout
+
+    assert 'Static report created' in output
```
Comment (abrugaro) [inline tests/analysis/java/test_selective_java_provider.py:19]: Can you make a call to `assert_story_points_from_report_file()` at the end of the test to ensure that the report is successfully created as the command states?

```suggestion
    assert 'Static report created' in output
    assert_story_points_from_report_file()
```

Comment (kpunwatk) [reply tests/analysis/java/test_selective_java_provider.py:19]: Missed that, thank You for noticing, Done!

**Review by sshveta — APPROVED** [review]

**Review by nachandr — APPROVED** [review]
LGTM

File: tests/analysis/java/test_selective_java_provider.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import os
+import subprocess
+
+from utils.command import build_analysis_command
```
Comment (abrugaro) [inline tests/analysis/java/test_selective_java_provider.py:4]: I suggested the call to the  method, but not the import

```suggestion
from utils.command import build_analysis_command
from utils.report import assert_story_points_from_report_file
```

**Review by abrugaro — CHANGES_REQUESTED** [review]

**Review by abrugaro — APPROVED** [review]

**Pull request merged**